app/controllers/declaration/RoutingCountriesController.scala

```diff
@@ -79,11 +79,10 @@ class RoutingCountriesController @Inject()(
     }
 
   private def redirectFromRoutingAnswer(mode: Mode, answer: Boolean)(implicit request: JourneyRequest[AnyContent]): Result =
-    mode match {
-      case ErrorFix if answer => Redirect(controllers.declaration.routes.RoutingCountriesController.displayRoutingCountry(mode))
-      case _ if answer        => navigator.continueTo(mode, controllers.declaration.routes.RoutingCountriesController.displayRoutingCountry)
-      case _                  => navigator.continueTo(mode, controllers.declaration.routes.LocationController.displayPage)
-    }
+    if (answer)
+      navigator.continueTo(mode, controllers.declaration.routes.RoutingCountriesController.displayRoutingCountry, mode == ErrorFix)
+    else
+      navigator.continueTo(mode, controllers.declaration.routes.LocationController.displayPage)
```

Comment:
The new parameter is required for this situation isn't it?  Here the mode might be ErrorFix but we are not passing isErrorFixInProgress=true (I presume that is right)

Reply:
It's not required, because if the answer is no we always will return to the Rejected notification page.